vue/components/organisms/ripe-image/ripe-image.vue

```diff
@@ -551,12 +551,15 @@ export const RipeImage = {
             curve: this.curve
         });
 
+        this.image.bind("error", () => this.$emit("error"));

```

Comment:
Don't you have to `unbind` as well?

Reply:
It's not being done in configurator as well

Reply:
Well it should probably be then